### PR TITLE
bump: gpu base image

### PR DESCRIPTION
**Pull request by alexeldeib**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Comment by alexeldeib** [comment]
to bump the base image from mcr - https://github.com/Azure/aks-gpu/pull/2/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R3

File: parts/linux/cloud-init/artifacts/cse_helpers.sh [hunk 1]
```diff
@@ -96,7 +96,7 @@ export GPU_DEST=/usr/local/nvidia
 NVIDIA_DOCKER_VERSION=2.8.0-1
 DOCKER_VERSION=1.13.1-1
 NVIDIA_CONTAINER_RUNTIME_VERSION="3.6.0"
-export NVIDIA_DRIVER_IMAGE_TAG="${GPU_DV}-v0.0.2-rev20-sha-a0bbd5"
+export NVIDIA_DRIVER_IMAGE_TAG="${GPU_DV}--sha-106e2e"
```
Comment (alexeldeib) [inline parts/linux/cloud-init/artifacts/cse_helpers.sh:99]: lol

Comment (alexeldeib) [reply parts/linux/cloud-init/artifacts/cse_helpers.sh]: double dash, fixed

Comment (ganeshkumar5699) [inline parts/linux/cloud-init/artifacts/cse_helpers.sh]: are there supposed to be two "--" here? The version in install-deps has one.

Comment (ganeshkumar5699) [reply parts/linux/cloud-init/artifacts/cse_helpers.sh]: oh, you just fixed it :)

**Review by anujmaheshwari1 — APPROVED** [review]

**Review by ganeshkumar5699 — APPROVED** [review]
Lgtm pending tests (other than current AB e2e, as it seems to have an unrelated setup failure)

**Pull request merged**